### ISSUE TITLE
exchanging gadget grid mode not work below blocks

### ISSUE THREAD

**Issue by redgoblin88** [Direwolf20-MC/BuildingGadgets#426]
TLDR:

upon making a "raw" instance (only forge and BG, nothing else) and trying to use the exchanging gadget on grid mode, attempted use on the bottom of the blocks doesnt work

on a fresh instance with only building gadgets (versions listed below), get an exchanging gadget, make a small platform in the sky (something like a 5x5 or 7x7... a decent size you know?) and set the exchanging gadget to grid mode and to a different block.

Note how on the top block and any of the sides the gadget will render what blocks are to be replaced and sucesfully replace them, but on the bottom of the platform nothing will render and attempting to exchange will fail (as seen in the attached screenshots)

screenies:
![2020-01-05_09 50 42](https://user-images.githubusercontent.com/13396472/71778189-802f1c80-2fa2-11ea-935d-5e5d0c7a46cb.png)
![2020-01-05_09 50 47](https://user-images.githubusercontent.com/13396472/71778191-81f8e000-2fa2-11ea-847e-b590f9812f3d.png)

mods:
Forge: 14.23.5.2847 (1.12.2)
Building gadgets: 2.74

**Comment by MichaelHillcox** [comment]
This is known and will be fixed in a back port. Thanks for the bug report though ❤️

**Comment by mkaito** [comment]
Hate to be that guy bumping old issues, but was this actually backported and released for 1.12.2?
